src/main/java/com/tileman/TilemanModePlugin.java

```diff
@@ -247,42 +247,57 @@ int getRemainingTiles() {
         return remainingTiles;
     }
 
-    void markTile(LocalPoint localPoint, boolean menuOption) {
-        if (localPoint == null || client.isInInstancedRegion() || (lastTile != null && lastTile.distanceTo(localPoint) == 0)) {
```

Comment:
Should we still not check the tile is different with
`(lastTile != null && lastTile.distanceTo(localPoint) == 0)`